invenio_requests/assets/semantic-ui/js/invenio_requests/search/MobileRequestItem.js

```diff
@@ -76,16 +76,19 @@ export const MobileRequestItem = ({
             {creatorName}
           </small>
           <small className="block rel-mt-1">
-            {result.receiver.community && result.expanded?.receiver.metadata.title && (
-              <>
-                <Icon
-                  className="default-margin"
-                  name={getUserIcon(result.expanded?.receiver)}
-                />
-                <span className="ml-5">{result.expanded?.receiver.metadata.title}</span>
-                {result.expires_at && " - "}
-              </>
-            )}
+            {result.receiver?.community &&
+              result.expanded?.receiver?.metadata.title && (
+                <>
+                  <Icon
+                    className="default-margin"
+                    name={getUserIcon(result.expanded?.receiver)}
+                  />
+                  <span className="ml-5">
+                    {result.expanded?.receiver.metadata.title}
```

Comment:
minorest of nits, but we know we have this here

```suggestion
                    {result.expanded.receiver.metadata.title}
```